### DISCUSSION TITLE
Make the editor box fill the container on focus

### DISCUSSION THREAD

**Discussion by timbressingtonboxuk** [ueberdosis/tiptap#1546] [General Questions] [answered]
Apologies if this has been asked previously but I'm wondering if there's a way of making the tiptap editor stretch the full height of it's container, rather than being on a single line, so that I can click anywhere inside the container to access the editor?

I've included a screenshot to illustrate, so rather than clicking the single line to enter the editor I would like the entire container to be clickable to enter the editor.

![image](https://user-images.githubusercontent.com/48092673/124614905-a5893900-de6c-11eb-8889-1dc429548192.png)

**Comment by cadars** [comment, marked as answer]
You can do that with the CSS flex property, with `flex-direction: column;` and `flex: 1` on the container.
